events/processor/index.js

```diff
@@ -0,0 +1,77 @@
+import AWS from 'aws-sdk';
+import uuid from 'uuid';
+
+import Echo from './types/echo';
+
+export default class Processor {
+  constructor(event) {
+    this.kinesis = new AWS.Kinesis();
+    this.event = event;
+    this.types = {
+      'echo': Echo
```

Comment:
I've been burned enough times by not quoting object keys that I just quote them all. [ESLint](http://eslint.org/docs/2.0.0/rules/quote-props) throws an error on unquoted object keys by default as well, which makes me think it's probably best practice.